components/infobox/commons/infobox_show.lua

```diff
@@ -25,59 +32,40 @@ function Show:createInfobox()
 	local infobox = self.infobox
 	local args = self.args
 
-	infobox:name(args.name)
-	infobox:image(args.image, args.defaultImage)
-	infobox:centeredCell(args.caption)
-	infobox:header('Show Information', true)
-	infobox:fcell(Cell:new('Organizer')
-				:options({})
-				:content(
-					unpack(self:_createHosts(args))
-				)
-				:make()
-			)
-	infobox:cell('Format', args.format)
-	infobox:cell('Airs', args.airs)
-	infobox:fcell(Cell:new('Location'):options({}):content(
-			self:_createLocation(args.country, args.city),
-			self:_createLocation(args.country2, args.city2)
-		):make())
-	self:addCustomCells(infobox, args)
-
-	local links = Links.transform(args)
-	infobox:header('Links', not Table.isEmpty(links))
-	infobox:links(links)
-	infobox:centeredCell(self:_addSecondaryLinkDisplay(args))
-	self:addCustomContent(infobox, args)
-	infobox:centeredCell(args.footnotes)
-	infobox:bottom(self:createBottomContent(infobox))
+	local widgets = {
+		Header{name = args.name, image = args.image},
+		Center{content = {args.caption}},
+		Title{name = 'Show Information'},
+		Cell{name = 'Host(s)', content = self:getAllArgsForBase(args, 'host', {makeLink = true})},
+		Cell{name = 'Format', content = {args.format}},
+		Cell{name = 'Airs', content = {args.airs}},
+		Cell{name = 'Location', content = {
+				self:_createLocation(args.country, args.city),
+				self:_createLocation(args.country2, args.city2)
+			}},
+		Customizable{id = 'custom', children = {}},
+		Builder{
+			builder = function()
+				local links = Links.transform(args)
+				local secondaryLinks = Show:_addSecondaryLinkDisplay(args)
+				if (not Table.isEmpty(links)) or (secondaryLinks ~= '') then
```

Comment:
This will crash if `links` equals nil and `secondaryLinks` doesn't.

Reply:
should be okay now (i hope)